test/test_other.py

```diff
@@ -14285,23 +14285,49 @@ def test_pthread_kill(self):
   @node_pthreads
   @parameterized({
     '': ([], 1),
+    'growable_arraybuffers': (['-sGROWABLE_ARRAYBUFFERS', '-Wno-experimental'], 1),
     'proxy': (['-sPROXY_TO_PTHREAD', '-sEXIT_RUNTIME'], 2),
   })
   def test_pthread_growth_mainthread(self, cflags, pthread_pool_size):
+    if '-sGROWABLE_ARRAYBUFFERS' in cflags:
+      self.node_args.append('--experimental-wasm-rab-integration')
+      self.require_node_canary()
+    else:
+      self.cflags.append('-Wno-pthreads-mem-growth')
```

Comment:
Nit: I'd keep this inline in the original flags, after all if that warning shows up in "growable array buffer" mode, we also want the test to error out as it would be unexpected.

Reply:
I'm not sure I understand.  The whole point of moving this flag so that it is not specified in "growable_arraybuffers" mode is so that if that warning ever does show up in that mode the test would error.

If we unconditionally specify `-Wno-pthreads-mem-growth` we would never see that error or warning since that flag disables it.

Reply:
Yup, my bad - I mixed up `-Wno-` with `-Werror-` 🤦‍♂️